lib/cartopy/io/img_tiles.py

```diff
@@ -32,29 +32,55 @@
 
 
 class GoogleWTS(metaclass=ABCMeta):
-    """
-    Implement web tile retrieval using the Google WTS coordinate system.
+    _MAX_THREADS = 24
 
-    A "tile" in this class refers to the coordinates (x, y, z).
+    def __init__(self,
+                 desired_tile_form='RGB',
+                 user_agent=f'CartoPy/{cartopy.__version__}',
+                 cache=False,
+                 resolution="",
+                 layer=None,
+                 style=None):
```

Comment:
In the cache directory implementation you use the ordering (layer, style, resolution), for consistency we should keep things here the same as well then. We might as well put these after a keyword only flag too.
```suggestion
                 *,
                 layer=None,
                 style=None
                 resolution="",):
```